app/templates/email/config.ts

```diff
@@ -0,0 +1,2 @@
+export const BAFU_VISUALIZE_EMAIL = "visualize@bafu.admin.ch"
+export const IXT_SUPPORT_EMAIL = "support@interactivethings.com"
```

Comment:
Thanks @noahonyejese for adding this 🙇‍♂️

A small note on the naming: with making this more generic, in case it's deployed in other contexts, maybe you could use something like `OWNER_ORGANIZATION_EMAIL` and `SUPPORT_EMAIL` (so it's not tied to our name and BAFU name)?

I was also thinking about a JSON config file, so it would be easier to update the values, but it's also fine like this 👍